src/java.desktop/share/classes/sun/awt/shell/ShellFolderManager.java

```diff
@@ -86,44 +86,12 @@ public Object get(String key) {
             // folders, such as Desktop, Documents, History, Network, Home, etc.
             // This is used in the shortcut panel of the filechooser on Windows 2000
             // and Windows Me
-            return checkFiles(new File[] { (File)get("fileChooserDefaultFolder") });
+            return new File[] { (File)get("fileChooserDefaultFolder") };
         }
 
         return null;
     }
 
-    private static File checkFile(File f) {
-        @SuppressWarnings("removal")
-        SecurityManager sm = System.getSecurityManager();
-        return (sm == null || f == null) ? f : checkFile(f, sm);
-    }
-
-    private static File checkFile(File f, @SuppressWarnings("removal") SecurityManager sm) {
-        try {
-            sm.checkRead(f.getPath());
-            if (f instanceof ShellFolder) {
-                ShellFolder sf = (ShellFolder)f;
-                if (sf.isLink()) {
-                    sm.checkRead(sf.getLinkLocation().getPath());
-                }
-            }
-            return f;
-        } catch (SecurityException | FileNotFoundException e) {
-            return null;
-        }
-    }
-
-    private static File[] checkFiles(File[] fs) {
-        @SuppressWarnings("removal")
-        SecurityManager sm = System.getSecurityManager();
-        return (sm == null || fs == null) ? fs : checkFiles(Stream.of(fs), sm);
-    }
-
-    private static File[] checkFiles(Stream<File> fs, @SuppressWarnings("removal") SecurityManager sm) {
```

Comment:
Minor: Similar to what @azvegint pointed out in Win32ShellFolderManager.java `import java.util.stream.Stream;`  is unused after removal of this method.

Reply:
fixed